### DISCUSSION TITLE
Error searching messages, status 401

### DISCUSSION THREAD

**Discussion by PamPans** [victornpb/undiscord#681] [1. Questions & Answers]
Hi!
I just installed undiscord a few hours agp. I've made sure to use the latest version of Tampermonkey, tun on developer mode and i open discord in my browser(chrome). When i open discord the trash-can icon is visible, upon pressing it i get met with the typical page (see picture)
![image](https://github.com/user-attachments/assets/378e70e1-5f66-4a8a-ba60-f034fa6d3a89)

I then press the button on the right to 'Author-, Server-, and Channel-ID'. The ID's are then loaded in and i press the 'Delete button'.
It runs for about 1 second and then shows error mesage, status 404! See underlaying image.
![image](https://github.com/user-attachments/assets/d4d1f443-778c-4de7-ac52-08c2a634450d)
(Also here is the message copied:  Error searching messages, API responded with status 401!
	{"message":"401: Unauthorized","code":0}
CoreException	{} ) 

Even when i have NSFW enabled, since allot of curse-words and so on have been used in the DM, it gives the same error message.
(see image
![image](https://github.com/user-attachments/assets/bf08ee7c-2d6a-4a63-b0dd-5a3d67e33b80)
)
This led me to github, where ive searched for people with a problem like mine, i found a thread speaking about this error code, that was already answered (https://github.com/victornpb/undiscord/discussions/280) this one is closed and the answer to the question was due to some updates/mantenance on discord happening Undiscord might not be working. i checked the site (https://discordstatus.com/ 11dec 2024) and i can see no such thing. 

So now im wondering what might specifically be wrong with mine? i ve been searching everywhere i can and simply cannot find a solution. I hope that the insight i gave on my situation was helpfull for whomever might wanna answer, and hope that you all have an amazing holiday! -. PamPans

**Comment by prodddlukeee** [comment]
I have this same problem, please if anyone knows how to fix it let me know.

**Comment by tatitatai** [comment]
same issue.

**Comment by bekkibau** [comment]
Fixed this issue on my fork for Chrome if you still need https://github.com/bekkibau/deletecord
